### ISSUE TITLE
Sftp Folders Mounted dont appeard 

### ISSUE THREAD

**Issue by Efeyabel** [microsoft/vscode#78786]
- VSCode Version: 1.37
- OS Version: Win10

Steps to Reproduce:

1. Mount folder vis sftp

I just upload to 1.37 vesion and now i cant see my sftp folders mounted via Netdrive. In the past versions i see it normally. If i put the letter in the explorer bar i can reach the folder.

I just check in another pc and the same. 

Img here.: [https://prnt.sc/oqh6dm](url)

**Comment by Efeyabel** [comment]
i can check this happends when i got "Execute as admin" the aplication

**Comment by bpasero** [comment]
@Efeyabel to clarify, you can open a folder but it shows up empty? Can you post a screenshot?

**Comment by chaoticgoo** [comment]
I am having a related problem with the July 2019 build that just installed today on my machine. I'll note it here instead of opening a new issue.

I use Ubuntu 16.04 desktop. I have an Ubuntu 16.04 VirtualBox VM running, and on the host I have folders mounted from the VM using SSHFS (like SFTP). I regularly open these with VS Code's Open Folders, e.g. under a path like this on my host: /media/vmname/somefolder/myproject

This has worked without a problem until today's update. Now I can still open my project folders, but suddenly some of their subfolders will not expand in the File Manager treeview. 

In each case the subfolder has the same name as the parent folder, but I doubt that's the problem. That is, if I open /media/vmname/somefolder/myproject, and it contains a subfolder also called myproject, that is the one that won't expand. Might be a coincidence.

In each case these subfolders also have one file that is a symlink within the VM but the Linux host thinks belongs to the host, and it can't be followed. I suspect this is what is choking VS Code, it can't walk the subfolder so I can see how the attempt to render the subfolder tree in the File Manager pane would die. 

So the triangle changes to point down like it expanded an empty folder, but it is not empty and my real file managers outside VS Code can show the file contents, and so can the VS Code terminal if i type `ls` in there. It's just the treeview that chokes.

However it worked until now.

````
Version: 1.37.0
Commit: 036a6b1d3ac84e5ca96a17a44e63a87971f8fcc8
Date: 2019-08-08T01:24:14.598Z
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 4.4.0-148-generic
```

**Comment by chaoticgoo** [comment]
Here's a screenshot showing the folder not-open in the treeview but a file listing working in VS Code terminal, and the OS file manager on the right has the same folder open no problem.

![2019-08-09_vscode_issue](https://user-images.githubusercontent.com/39128010/62816169-3294ad80-baf1-11e9-9beb-de257557ded9.png)

Excuse the blacked-out redactions. Hope the too-many Look Here feature arrows help.

Good luck and thank you!

**Comment by chaoticgoo** [comment]
Just made a test project folder called 'project' with a nested folder also called 'project'. Put files in each and all is well in VS Code with it. So the issue is not my really bad, repetitive folder naming conventions, it handles those fine. 

Note that this project **is** over SSHFS just like the real ones that are not working.

![2019-08-09_vscode_issue_not_the_folder_name](https://user-images.githubusercontent.com/39128010/62816282-7dfb8b80-baf2-11e9-8bfd-d6273f6f3080.png)

So I suspect it's that unfollow-able (from here on the host) symlink (config.json in the first screenshot above). I'll test that next.

**Comment by chaoticgoo** [comment]
Yep it's the remote symlink that is causing the treeview to fail to render the subtree. I moved `config.json` up one folder to the project root, reopened VS Code, and then the entire project tree fails to render.

![2019-08-09_vscode_issue_is_the_remote_symlink](https://user-images.githubusercontent.com/39128010/62816368-fc0c6200-baf3-11e9-958c-e106ea0007aa.png)

Then I moved the symlink out of my project (not an actual solution) and the treeview renders fine.

![2019-08-09_vscode_no_issue_if_no_remote_symlink](https://user-images.githubusercontent.com/39128010/62816381-4db4ec80-baf4-11e9-81bf-f074bb7ba1af.png)

I hope that's a smoking gun for you guys and gals and helps you solve it. 

Thanks again!
